src/configs/axios.config.ts

```diff
@@ -11,3 +11,22 @@ export const axiosInstance = axios.create({
     Authorization: `Bearer ${AccessTokenStore.getToken()}`,
   },
 });
+
+axiosInstance.interceptors.response.use(
+  (response) => response,
+  async (error) => {
+    const originalRequest = error.config;
+
+    if (error.response && error.response.status === 401) {
+      try {
+        await axiosInstance.get('/auth/refresh-token');
```

Comment:
lets use constant that you created for endpoints

Reply:
fixed